### PR TITLE
chore: fixes to release process after yarn switch

### PR DESCRIPTION
**Pull request by edvald**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -45,8 +45,7 @@ orbs:
       branches:
         ignore: /.*/
       tags:
-        # Technically could be a better regex but for now just starting with v feels good enough
-        only: /v.*/
+        only: /[0-9].*/
```
Comment (edvald) [inline .circleci/config.yml:48]: Does this make sense? This translates to any string starting with a number, basically.

Comment (edvald) [reply .circleci/config.yml:48]: I guess it's kinda fine, we don't use tags much for anything else.

File: scripts/release.ts [hunk 2]
```diff
@@ -129,8 +135,14 @@ async function release() {
...
-    "core/package.json", "core/yarn.lock",
-    "dashboard/package.json", "dashboard/yarn.lock",
+    "package.json",
+    "core/package.json",
+    "cli/package.json",
+    "dashboard/package.json",
+    "sdk/package.json",
+    "plugins/conftest-container/package.json",
+    "plugins/conftest-kubernetes/package.json",
+    "plugins/conftest/package.json"
```
Comment (edvald) [inline scripts/release.ts:145]: We should automatically create this list from the `packages` variable.

File: .circleci/config.yml (hunk 1 above)
Comment (thsig) [reply .circleci/config.yml:48]: Yeah. This really should be `/[0-9]+\.[0-9]+\.\[0-9]+(-[0-9]+)?` or the like.

**Comment by thsig** [comment]
Closing in favor of the release branch (which includes the changes made here).

**Pull request closed without merging**